### ISSUE TITLE
Allow the creation of posts

### ISSUE THREAD

**Issue by Ategon** [PangoraWeb/pangora-ui#3]
Should have all of the same features as lemmy-UI as well as the ability to enter alt text. If the user is not logged in should send them to the sign up page
